js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
There are basically two errors in that code: 1) We shouldn't save name on every `projectChanged`, because it could be something else that changed. 2) `.setSlug()` shouldnt be called with `project.name`, because name & slug are two different things.